src/components/LoginPage/LoginPageContainer.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import LoginPage from './LoginPage';
+import { userActions } from '../../actions';
+
+
+const mapStateToProps = state => ({
+  loggingIn: state.authentication.loggingIn,
```

Comment:
Use lodash's get to set default in `mapStateToProps` itself instead of doing it in the component.